provider/lxd.go

```diff
@@ -26,6 +26,8 @@ import (
 
 var _ execution.ExternalProvider = &LXD{}
 
+var version = "v0.1.0"
```

Comment:
This is not the interface version. This is the provider version.... The `GetVersion()` command gets the **provider** version. Not the interface version.

This needs to be `0.0.0-unknown` by default. This will then be set by:

https://github.com/cloudbase/garm-provider-lxd/blob/b63472c5becad2b9a39603e37f67f26d361e0d30/scripts/build-static.sh#L39

which also needs to be updated to no longer try to set the `Version` in main.go.

Also, this variable **must** be exported.